api/src/main/java/org/pmiops/workbench/db/model/DbUser.java

```diff
@@ -243,10 +244,6 @@ public Timestamp getFirstRegistrationCompletionTime() {
     return firstRegistrationCompletionTime;
   }
 
-  public void setFirstRegistrationCompletionTime() {
```

Comment:
unused - only the version with a parameter was used.  This version would be tricky to test because we couldn't use our usual FakeClock